inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -102,15 +102,20 @@ GNAPluginNS::HeaderLatest::ModelHeader GNAModelSerial::ReadHeader(std::istream &
 
     is.seekg(0, is.beg);
     Header2dot1::ModelHeader tempHeader2dot1;
+    Header2dot3::ModelHeader tempHeader2dot3;
     switch (header.version.major) {
         case 2:
             switch (header.version.minor) {
```

Comment:
before break in this switch case make:
readNbytes(&header, header_byte_size, is)